### ISSUE TITLE
Docker-comspse memory settings cause coordinator OOM

### ISSUE THREAD

**Issue by ivansenic** [stargate/jsonapi#414]
Memory ops for the coordinator in the docker compose scrips are allowing the max heap size to be same as the max memory given to the container, causing the OOM kill. Off heap memory is not taken into account.. 

https://github.com/stargate/jsonapi/blob/6473f8723d1e80da34a0a95b68cb8da991e8e2b1/docker-compose/docker-compose.yml#L75-L77

We should go with Quarkus memory settings here, and set these env vars:

```
JAVA_MAX_MEM_RATIO: '75'
JAVA_INITIAL_MEM_RATIO: '50'
```

**Comment by jeffreyscarpenter** [comment]
Do we need to do similar changes in docker compose files in `stargate/stargate`?

**Comment by ivansenic** [comment]
> Do we need to do similar changes in docker compose files in `stargate/stargate`?

Good point, seems we do. Created https://github.com/stargate/stargate/issues/2570
